python/test/t_Waarts_25_quadratic_terms.expout

```diff
@@ -1201,10 +1201,10 @@ CI at 90% =[ 6.04093e-04 ; 6.14696e-03 ]
 ************************************************************************************************
 ******************************************* L H S **********************************************
 ************************************************************************************************
-Pf estimation = 4.62500e-03
-Pf Variance estimation = 5.75451e-07
-CoV = 0.16402
-90% Confidence Interval = 2.49552e-03
-CI at 90% =[ 3.37724e-03 ; 5.87276e-03 ]
+Pf estimation = 3.62500e-03
```

Comment:
This particular result looks suspicious to me because it seems that there is no significant digit in the estimated probability. Is the _exact_ probability known in this particular example? Whatever the reason, a probability lower than, say, $10^{-2}$, should not be used in a unit test of this algorithm with the LHS experiment: it is too close to zero to get the fast estimate that we need in a unit test.